src/mlpack/methods/ann/layer/isru.hpp

```diff
@@ -0,0 +1,123 @@
+/**
+ * @file methods/ann/layer/isru.hpp
+ * @author Suvarsha Chennareddy
+ *
+ * Definition of the ISRLU and ISRU activation function as described by Jonathan T. Barron.
```

Comment:
This implements the ISRU function right? Because ISRLU is already implemented see - https://github.com/mlpack/mlpack/blob/master/src/mlpack/methods/ann/layer/isrlu_impl.hpp.

Reply:
Yes, but the paper also describes the ISRU function in page 6.
Oh wait...unless you mean the implantation itself, yes it only implements the ISRU function. I will remove "ISRLU"